source/docs/articles/sites/mime-types.md

```diff
@@ -0,0 +1,22 @@
+---
+title: MIME Types
+description: Learn how to add, change, and remove a file extension to a MIME type on Pantheon.
+keywords: mime types, file extension, IANA, mime
+---
```

Comment:
Let's add an intro paragraph:

Pantheon promotes and respects internet standards, and for MIME Types we treat the [IANA Media Type List](http://www.iana.org/assignments/media-types/media-types.xhtml) as the canonical source of what headers to serve for what file extensions. This doc page describes what to do if you have a file type that isn't handled by IANA, or require an alternative.